openeo/processes.py

```diff
@@ -1808,6 +1808,18 @@ def predict_random_forest(self, model) -> 'ProcessBuilder':
         """
         return predict_random_forest(data=self, model=model)
 
+    def predict_catboost(self, model) -> 'ProcessBuilder':
+        """
+        Predict values from a Catboost model
+
+        :param self: An array of numbers.
+        :param model: A model object that can be trained with the processes ``fit_regr_catboost()``
+            (regression) and ``fit_class_catboost()`` (classification).
+
+        :return: The predicted value.
+        """
+        return predict_catboost(data=self, model=model)
+
```

Comment:
openeo/processes.py is a generated file 
https://github.com/Open-EO/openeo-python-client/blob/6f0a86517389c4b3b48309c9c2c37f675b714587/openeo/processes.py#L2-L3

so it's nor recommended to manually add processes here (they will be removed with the next update)

Reply:
as long as `predict_catboost` is not an official process, we can not put it here